src/graphql/query/lenderPublicProfile.graphql

```diff
@@ -4,6 +4,7 @@ query LenderPublicProfile ($publicId: String!) {
 			id
 			name
 			loanCount
+			memberSince
 			image {
 				id
 				url
```

Comment:
The default url gives a 100x100 sized image, so let's change this to `url(customSize: "w200h200")` to match the size in the old-stack lender page.

Reply:
Done.